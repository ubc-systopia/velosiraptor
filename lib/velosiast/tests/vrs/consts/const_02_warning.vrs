const FOO: int = 1234;
const BAR: bool = true;
const foo: int = 1234;
const _FOO : int = 4567;
const _foo : int = 6765;