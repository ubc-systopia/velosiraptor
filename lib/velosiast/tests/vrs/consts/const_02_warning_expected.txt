VelosiAst(tests/vrs/consts/const_02_warning.vrs)
--------------------------------------------------------

const FOO : int = 0x4d2;
const BAR : bool = true;
const foo : int = 0x4d2;
const _FOO : int = 0x11d7;
const _foo : int = 0x1a6d;
--------------------------------------------------------

########################################################

warning: identifier `foo` should have an upper case name
     --> tests/vrs/consts/const_02_warning.vrs:3:7
      |
    3 |  const foo: int = 1234;
      |        ^^^ convert the identifier to upper case (notice the capitalization): `FOO`
warning: identifier `_foo` should have an upper case name
     --> tests/vrs/consts/const_02_warning.vrs:5:7
      |
    5 |  const _foo : int = 6765;
      |        ^^^^ convert the identifier to upper case (notice the capitalization): `_FOO`
warning: generated 2 warnings
